### PR TITLE
fix old assertion format serialization

### PR DESCRIPTION
**Pull request by mathnogueira**
This PR fixes the assertion format serialization

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: server/model/json.go [hunk 1]
```diff
@@ -76,10 +76,30 @@ func (a *Assertion) UnmarshalJSON(data []byte) error {
...
+		}
+
+		aux.Attribute = oldAssertion.Attribute
+		aux.Comparator = oldAssertion.Comparator
+		aux.Value = &AssertionExpression{
+			LiteralValue: LiteralValue{
+				Type:  "string",
+				Value: oldAssertion.Value,
+			},
+		}
```
Comment (mathnogueira) [inline server/model/json.go:102]: This solves the deserialization problem

File: server/model/json_test.go [hunk 2]
```diff
@@ -214,3 +214,26 @@ func TestRunEncoding(t *testing.T) {
...
+	oldAssertionJson, err := json.Marshal(oldAssertion)
+	require.NoError(t, err)
+
+	err = json.Unmarshal(oldAssertionJson, &newAssertion)
+	require.NoError(t, err)
+
+	assert.Equal(t, oldAssertion.Attribute, newAssertion.Attribute.String())
+	assert.Equal(t, oldAssertion.Comparator, newAssertion.Comparator.String())
+	assert.Equal(t, oldAssertion.Value, newAssertion.Value.String())
+}
```
Comment (mathnogueira) [inline server/model/json_test.go:239]: This reproduces and ensures the problem @jorgeepc found is no longer happening.

**Review by mathnogueira — COMMENTED** [review]
This is the same code from #1007 but with an important deserialization fix. I commented the points that need review

**Review by schoren — APPROVED** [review]
LGTM

**Pull request merged**